开始训练...
训练集数据处理。。。。
=============
验证集数据处理。。。。。
=============
0
D:\Users\Admin\Python37\lib\site-packages\paddle\nn\layer\norm.py:641: UserWarning: When training, we now always track global mean and variance.
  "When training, we now always track global mean and variance.")
Traceback (most recent call last):
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 114, in <module>
    train(model, EPOCH_NLM, opt)
  File "D:/Users/Admin/PycharmProjects/PyCharm_Projects/Deep_learning/cat_dog/model_training02.py", line 42, in train
    loss = fluid.layers.cross_entropy(logits, label)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\layers\loss.py", line 260, in cross_entropy
    return cross_entropy2(input, label, ignore_index)
  File "D:\Users\Admin\Python37\lib\site-packages\paddle\fluid\layers\loss.py", line 281, in cross_entropy2
    ignore_index)
ValueError: (InvalidArgument) Tensor holds the wrong type, it holds float, but desires to be int64_t.
  [Hint: Expected valid == true, but received valid:0 != true:1.] (at C:\home\workspace\Paddle_release\paddle/fluid/framework/tensor_impl.h:33)
  [operator < cross_entropy2 > error]
